.gitignore

```diff
@@ -2,6 +2,7 @@
 *.build.log
 .DS_Store
 /common/last-install.log
+/common/npm-local
```

Comment:
True, these are related to the version of NPMX you are using... This path was updated in the latest NPMX